testsuite/geb-testsuite/src/test/java/GebConfig.groovy

```diff
@@ -0,0 +1,133 @@
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+/*
+ This is the Geb configuration file.
+
+ See: http://www.gebish.org/manual/current/configuration.html
+*/
+
+import java.util.logging.Level
+
+import geb.driver.SauceLabsDriverFactory
+import org.openqa.selenium.Dimension
+import org.openqa.selenium.Platform
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.chrome.ChromeDriver
+import org.openqa.selenium.firefox.FirefoxDriver
+import org.openqa.selenium.ie.InternetExplorerDriver
+import org.openqa.selenium.phantomjs.PhantomJSDriver
+import org.openqa.selenium.phantomjs.PhantomJSDriverService
+import org.openqa.selenium.remote.DesiredCapabilities
+import org.openqa.selenium.remote.RemoteWebDriver
+import org.openqa.selenium.safari.SafariDriver
+
+reportsDir = "target/test-reports/geb"
+// Port is not known until runtime, needs to be set in BaseTest.groovy
+//baseUrl = System.getProperty('geb.build.baseUrl', 'http://localhost:9070/')
+
+//enable waitFor behaviour for all 'at' checks
+atCheckWaiting = true
+waiting {
+  presets {
+    slow {
+      timeout = 15
+    }
+  }
+}
+// Consult these documents for how to configure for SauceLabs
+// http://www.gebish.org/manual/current/sauce-labs.html#saucelabs_integration
+// https://saucelabs.com/docs/platforms/webdriver
+def sauceBrowser = System.getProperty("geb.sauce.browser")
+if (sauceBrowser) {
+  driver = {
+    def username = System.getProperty("GEB_SAUCE_LABS_USER")
+    assert username
+    def accessKey = System.getProperty("GEB_SAUCE_LABS_ACCESS_PASSWORD")
+    assert accessKey
+    WebDriver driver = new SauceLabsDriverFactory().create(sauceBrowser, username, accessKey)
+    driver.manage().window().maximize()
+    return driver
+  }
+  // increase default timeouts to account for remote execution
+  waiting {
+    timeout = 10
+    retryInterval = 0.5
+  }
+}
+else {
+  driver = { configure(new FirefoxDriver()) }
+}
+
+Platform current = Platform.current
+environments {
+
+  // run as “mvn -Dgeb.env=chrome test”
+  // See: http://code.google.com/p/selenium/wiki/ChromeDriver
+  chrome {
+    driver = { configure(new ChromeDriver()) }
+  }
+
+  // see https://code.google.com/p/selenium/wiki/SafariDriver
+  safari {
+    if (current.is(Platform.MAC)) {
+      driver = { configure(new SafariDriver()) }
+    }
+    else {
+      throw new IllegalStateException('Only runs on mac!')
+    }
+  }
+
+  // see https://code.google.com/p/selenium/wiki/InternetExplorerDriver
+  ie {
+    if (current.is(Platform.WINDOWS)) {
+      driver = { configure(new InternetExplorerDriver()) }
+    }
+    else {
+      throw new IllegalStateException('Only runs on windows!')
+    }
+  }
+
+  // see https://github.com/detro/ghostdriver
+  phantom {
+    String phantomJsBinary
+    new File(System.getProperty('geb.workdir', 'target') + '/phantomjs-maven-plugin').eachFileRecurse { File file ->
```

Comment:
This is the equivalent of the https://github.com/sonatype/nexus-oss/blob/gebTest/testsupport/nexus-siestajs-testsupport/src/main/java/org/sonatype/nexus/testsuite/siestajs/TestLauncher.groovy code, finding the unpacked dependency so that we can call the native executable to launch phantomjs